src/subdomains/generic/user/models/auth/auth.service.ts

```diff
@@ -61,13 +61,13 @@ export class AuthService {
     const existingUser = await this.userRepo.getByAddress(dto.address, true);
     if (existingUser) throw new ConflictException('User already exists');
 
-    const wallet = await this.walletRepo.findOneBy({ id: dto.walletId });
+    const wallet = await this.walletRepo.findOneBy([{ id: dto.walletId }, { name: dto.wallet }]);
```

Comment:
This is missing in UserService L118. Maybe add a parameter `wallet?: Wallet` to the `createUser` method? Then we don't need to search for the wallet again (except if it's null we take the default one).